README.md

```diff
@@ -139,6 +143,7 @@ The pipeline uses nf-core modules contributed by following authors:
 <a href="https://github.com/charles-plessy"><img src="https://github.com/charles-plessy.png" width="50" height="50"></a>
 <a href="https://github.com/bunop"><img src="https://github.com/bunop.png" width="50" height="50"></a>
 <a href="https://github.com/abhi18av"><img src="https://github.com/abhi18av.png" width="50" height="50"></a>
+<a href="https://github.com/liamlelievre"><img src="https://github.com/liamlelievre.png" width="50" height="50"></a>
```

Comment:
This section is automatically populated by parsing the `meta.yml` of modules/subworkflows. For example: https://github.com/Plant-Food-Research-Open/genepal/blob/ee702d71a9ea4f422c92729533ff0564c231b5e8/modules/nf-core/eggnogmapper/meta.yml#L82

Your name might get deleted unintentionally in a future update. That's why I have added it up top.

Reply:
Cool cheers